prog/weave-kube/weave-daemonset-k8s-1.6.yaml

```diff
@@ -47,6 +47,44 @@ items:
       - kind: ServiceAccount
         name: weave-net
         namespace: kube-system
+  - apiVersion: rbac.authorization.k8s.io/v1beta1
+    kind: Role
+    metadata:
+      name: weave-net2
```

Comment:
Can this be renamed to `weave-net`?

Reply:
(Also needs changes to the labels, and the rolebinding should be renamed, and reference the new name)

Reply:
@mikebryant I'm not sure this can be changed, as it would clash with the `weave-net`  `ClusterRole` which grants different permissions at a different scope.

Reply:
Why would it clash? They're two different types of resource, and names are only unique within a resource.

e.g.
```
(devtools/kube-system) ~ $ k get clusterrole mike
NAME      AGE
mike      9s
(devtools/kube-system) ~ $ k get role mike
NAME      AGE
mike      8s
```

Reply:
👍 I stand corrected. Changing now.